sdks/python/apache_beam/runners/portability/fn_api_runner/fn_runner_test.py

```diff
@@ -136,6 +136,37 @@ def test_batch_pardo(self):
 
       assert_that(res, equal_to([6, 12, 18]))
 
+  def test_batch_pardo_override_type_inference(self):
+
+    class ArrayMultiplyTransposedDoFn(beam.DoFn):
```

Comment:
I might be misunderstanding something, but why is "Transposed" in the name?

Reply:
Whoops. I was originally trying something fancy here but reverted it. I'll change it.